controller_manager_tests/CMakeLists.txt

```diff
@@ -15,18 +15,13 @@ catkin_package(
 
 #common commands for building c++ executables and libraries
 add_library(${PROJECT_NAME}
-  src/my_robot_hw.cpp
-  include/controller_manager_tests/my_robot_hw.h
   src/effort_test_controller.cpp
-  include/controller_manager_tests/effort_test_controller.h
-  src/vel_eff_controller.cpp
-  include/controller_manager_tests/vel_eff_controller.h
+  src/extensible_controllers.cpp
+  src/my_dummy_controller.cpp
+  src/my_robot_hw.cpp
   src/pos_eff_controller.cpp
-  include/controller_manager_tests/pos_eff_controller.h
   src/pos_eff_opt_controller.cpp
-  include/controller_manager_tests/pos_eff_opt_controller.h
-  src/my_dummy_controller.cpp
-  include/controller_manager_tests/my_dummy_controller.h
```

Comment:
Separate PR for this change in #293.

Reply:
Merged, please rebase this one without that commit.

Reply:
Done.